pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -600,13 +600,14 @@ func (jm *ControllerV2) syncCronJob(
 	}
 	cj.Status.Active = append(cj.Status.Active, *jobRef)
 	cj.Status.LastScheduleTime = &metav1.Time{Time: *scheduledTime}
-	if _, err := jm.cronJobControl.UpdateStatus(cj); err != nil {
+	updatedCJ, err = jm.cronJobControl.UpdateStatus(cj)
```

Comment:
I was confused as the `cj` is a `reference type`,  whether this  will happen? correct me if not.

Reply:
It updates status by sending a http request to the apiserver, which doesn't change the value of `cj`.

Reply:
oh yeah, forgive me.

Reply:
another question, as the referred issue mentioned, with new created cronjob, this error will not occur. But if the root cause is what you said, I think this error will always happen, right?

Reply:
Actually,  yes. I reproduced this in my cluster.